lib/chef/resource/build_essential.rb

```diff
@@ -70,7 +71,7 @@ class BuildEssential < Chef::Resource
                 # in Apple's SUS catalog
                 touch /tmp/.com.apple.dt.CommandLineTools.installondemand.in-progress
                 # find the CLI Tools update. We tail here because sometimes there's 2 and newest is last
-                PROD=$(softwareupdate -l | grep "\*.*Command Line" | tail -n 1 | awk -F"*" '{print $2}' | sed -e 's/^ *//' | tr -d '\n')
+                PROD=#{update_label}
                 # install it
                 softwareupdate -i "$PROD" --verbose
```

Comment:
```suggestion
                softwareupdate -i "#{xcode_version}" --verbose
```

This is a bit unnecessarily verbose.